src/chromium/interoperability-with-cpp/error-handling.md

```diff
@@ -3,39 +3,21 @@
 cxx's support for `Result<T,E>` relies on C++ exceptions, so we can't use that
 in Chromium. Alternatives:
 
-* Where success can be represented as a simple Boolean, as done in our [QR code generator][1]:
-  Return a Boolean representing success, and record results using out-parameters:
-  ```rust,ignore
-  #[cxx::bridge(namespace = "qr_code_generator")]
-  mod ffi {
-    extern "Rust" {
-      fn generate_qr_code_using_rust(
-        data: &[u8],
-        min_version: i16,
-        out_pixels: Pin<&mut CxxVector<u8>>,
-        out_qr_size: &mut usize,
-      ) -> bool;
-    }
-  }
```

Comment:
Move this into a subsection - `src/chromium/interoperability-with-cpp/error-handling-qr.md`